### PR TITLE
resolved get_keystore unit test

### LINKED ISSUE #773: [BUG] get_keystore test using absolute path
**Describe the bug**
Get_keystone checks if the path contains the word 'config'. However, if we have a branch with a name including the word 'config', the get_keystone test will fail on Jenkins because it is taking the branch name to create a new folder and validate against the absolute path. We should not test against an absolute path because we should only care about what is within the opensearch dashboard application but not the path outside of opensearch dashboards.

**To Reproduce**
1. Create a branch and name it with the word 'config'
2. Go to Jenkins
3. Build
4. See error log under unit test: get_keystore test failed

**Screenshots**
Screen Shot 2021-09-08 at 6 26 11 PM

### PR DESCRIPTION
**Pull request by Aigerim-ai**
### Description
This commit fixes #773 

### Check List

- [x] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [x] New functionality includes testing.
- [x] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by Aigerim-ai** [comment]
@abbyhu2000  hi! can you please look at this commit?

**Comment by abbyhu2000** [comment]
> @abbyhu2000 hi! can you please look at this commit?

LGTM! Thank you @Aigerim-ai

**Review by abbyhu2000 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -192,8 +192,10 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Vis Builder] Adds field unit tests ([#3211](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3211))
 - [BWC Tests] Add BWC tests for 2.6.0 ([#3356](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3356))
 - Prevent primitive linting limitations from being applied to unit tests found under `src/setup_node_env` ([#3403](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3403))
+- [Tests] Resolves the issue with get_keystore.js test([#3854]https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3854)
```
Comment (AMoo-Miki) [inline CHANGELOG.md:195]:
```suggestion
- [Tests] Fix unit tests for `get_keystore` ([#3854]https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3854)
```

File: src/cli_keystore/get_keystore.test.js [hunk 2]
```diff
@@ -32,6 +32,8 @@ import { getKeystore } from './get_keystore';
 import { Logger } from '../cli_plugin/lib/logger';
 import fs from 'fs';
 import sinon from 'sinon';
+import { getConfigDirectory, getDataPath } from '@osd/utils';
+import { join } from 'path';
```
Comment (AMoo-Miki) [inline src/cli_keystore/get_keystore.test.js:36]: Will this work on Windows too?

File: CHANGELOG.md (hunk 1 above)
Comment (manasvinibs) [inline CHANGELOG.md:195]: nit: should be ```([#3854](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3854))```

**Review by AMoo-Miki — CHANGES_REQUESTED** [review]

**Comment by manasvinibs** [comment]
@abbyhu2000 Changes have to be rebased as there is a merge conflict and also DCO check is failing.

**Review by AMoo-Miki — DISMISSED** [review]

**Comment by AMoo-Miki** [comment]
The required commits have been signed-off; will pass DCO.

**Review by abbyhu2000 — APPROVED** [review]

**Review by manasvinibs — APPROVED** [review]

**Pull request merged**